android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetOngoingConferenceService.java

```diff
@@ -132,7 +159,7 @@ public void onCreate() {
             stopSelf();
             JitsiMeetLogger.w(TAG + " Couldn't start service, notification is null");
         } else {
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
```

Comment:
Why is this change necessary?

Reply:
Because minimum is now 30 for FOREGROUND_SERVICE_TYPE_MICROPHONE.